libtransmission/torrent.h

```diff
@@ -990,6 +1007,7 @@ struct tr_torrent
     libtransmission::SimpleObservable<tr_torrent*> swarm_is_all_upload_only_;
     libtransmission::SimpleObservable<tr_torrent*, tr_file_index_t const*, tr_file_index_t, tr_priority_t> priority_changed_;
     libtransmission::SimpleObservable<tr_torrent*, bool> sequential_download_changed_;
+    libtransmission::SimpleObservable<tr_torrent*, bool> sequential_download_from_piece_changed_;
```

Comment:
Missing some changes I requested before.

```suggestion
    libtransmission::SimpleObservable<tr_torrent*, tr_piece_index> sequential_download_from_piece_changed_;
```

P.S. Didn't notice this earlier because I didn't expect this to compile...